src/lib/i18n/languageKeys/keys/settings/channels/Logs.ts

```diff
@@ -0,0 +1,19 @@
+import { T } from '#lib/types';
+
+export const ChannelCreate = T<string>('settings:channelsLogsChannelCreate');
+export const ChannelDelete = T<string>('settings:channelsLogsChannelDelete');
+export const ChannelUpdate = T<string>('settings:channelsLogsChannelUpdate');
+export const EmojiCreate = T<string>('settings:channelsLogsEmojiCreate');
+export const EmojiDelete = T<string>('settings:channelsLogsEmojiDelete');
+export const EmojiUpdate = T<string>('settings:channelsLogsEmojiUpdate');
+export const Image = T<string>('settings:channelsLogsImage');
+export const Member = T<string>('settings:channelsLogsMember');
+export const Message = T<string>('settings:channelsLogsMessage');
+export const Moderation = T<string>('settings:channelsLogsModeration');
+export const NsfwMessage = T<string>('settings:channelsLogsNsfwMessage');
+export const Prune = T<string>('settings:channelsLogsPrune');
+export const Reaction = T<string>('settings:channelsLogsReaction');
+export const RoleCreate = T<string>('settings:channelsLogsRoleCreate');
+export const RoleDelete = T<string>('settings:channelsLogsRoleDelete');
+export const RoleUpdate = T<string>('settings:channelsLogsRoleUpdate');
+export const ServerUpdate = T<string>('settings:channelsLogsServerUpdate');
```

Comment:
Bunch of these are missing from `en-US/settings.json`. They are used in `s!conf`.

Reply:
Note that you can get the content from skyra.pw because generally the tooltip text was made to match what's in settings. But do not my review comments on there that changes those.

Reply:
Fixed in 6c6a6a06.